.github/workflows/golang.yml

```diff
@@ -67,10 +92,32 @@ jobs:
       uses: actions/setup-go@v5
       with:
         go-version: ${{ env.GOLANG_VERSION }}
```

Comment:
There is `cache: true` that might be able to save some time on the actions run below this